.gitignore

```diff
@@ -88,6 +88,7 @@ package-lock.json
 *.swp
 *.tmp
 *.bak
+*.map
```

Comment:
The new builds process creates `.map` files, like `block-lab/css/blocks.editor.css.map`.